### ISSUE TITLE
COB changes preferred labels of several ontologies

### ISSUE THREAD

**Issue by matentzn** [OBOFoundry/COB#243]
COB changes preferred labels of several ontologies, which is kinda "injection".

Examples:

id: GO:0005634
name: cell nucleus (instead of nucleus)
id: GO:0003674
name: gene product or complex activity (instead of [molecular function](https://github.com/OBOFoundry/COB/blob/master/cob-base.owl#L1620))
id: BFO:0000020
name: characteristic (instead of specifically dependent continuant)

This is related to the question of "who owns COB terms". 

I am beginning to question a bit if adopting URIs is really the right way in COB, vs just using clean new COB ids with sssom mappings..

**Comment by cmungall** [comment]
This was originally by design but doesn't make sense in light of the merge strategy

> I am beginning to question a bit if adopting URIs is really the right way in COB, vs just using clean new COB ids with sssom mappings..

we should pursue this. But for now I suggest:

- we switch to the authoritative labels for domain ontologies (possibly making issues on those ontologies trackers requesting a rename - good luck)
- in some cases we consider a non equivalent mapping in sssom-to-external
    - I don't think characteristic should be equivalent to SDC. I'm not sure there is even value in mapping to BFO

**Comment by balhoff** [comment]
> > renaming molecular function to "gene product or complex activity"
> 
> Is there any real hope here? :D

I proposed we rename "molecular_function" (the actual label) to "molecular function", and haven't gotten very far with that. 😂

**Comment by wdduncan** [comment]
To be honest, the ditching of COB ids was a surprise to me.

> I don't think characteristic should be equivalent to SDC. 

The definition of SDC requires that there exist at least one IC that the SDC `s-depends_on`:
```
b is a specifically dependent continuant = Def. b is a continuant & there is some independent continuant c which is not a spatial region and which is such that b s-depends_on c at every time t during the course of b’s existence. (axiom label in BFO2 Reference: [050-003])

(iff (SpecificallyDependentContinuant a) (and (Continuant a) (forall (t) (if (existsAt a t) (exists (b) (and (IndependentContinuant b) (not (SpatialRegion b)) (specificallyDependsOnAt a b t))))))) // axiom label in BFO2 CLIF: [050-003] 
```
A `cob:characteristic`  can be a characteristic of a process. But, can a `cob:characteristic` can depend on processes w/o necessarily depending on ICs? I supposed this depends on whether you accept the premise that`bfo:process` `s-depend_on` material entities:
```
p is a process = Def. p is an occurrent that has temporal proper parts and for some time t, p s-depends_on some material entity at t. (axiom label in BFO2 Reference: [083-003])

(iff (Process a) (and (Occurrent a) (exists (b) (properTemporalPartOf b a)) (exists (c t) (and (MaterialEntity c) (specificallyDependsOnAt a c t))))) // axiom label in BFO2 CLIF: [083-003] 
```
If you take `s-depends_on` to be transitive (in some sense), then characteristics of processes necessarily still `s-depend_on an` an IC. The BFO2 doc (page 54) has an axiom which (I think) supports this:
```
AXIOM: if b s-depends_on c at t & c s-depends_on d at t then b s-depends_on d at t. [054-002]
```
A question to address here is whether `characteristic of` is the same as `s-depends_on`.  
The definition for `characteristic of` is:
```
a relation between a specifically dependent continuant (the characteristic) and any other entity (the bearer), in which the characteristic depends on the bearer for its existence.
```
The definition of `s-depends_on` (page 23, BFO doc) is:
```
ELUCIDATION: To say that b s-depends_on c at t is to say that 	
  b and c do not share common parts 
  & b is of a nature such that it cannot exist unless c exists 
  & b is not a boundary of c and b is not a site of which c is the host [64]. [012-002]]
```
The definitions for `characteristic of` and `s-depends_on` seem to have a lot of similarities. But, perhaps one key difference is that `characteristic of` does not exclude characteristics of boundaries and sites. I don't understand the reasoning for this exclusion in BFO. It seems reasonable to me that sites can have characteristics such being a certain size and shape. But, there may be something crucial I don't understand about sites. 

To sum up, I don't think characteristics are the same as SDCs if:
1. COB allows for characteristics of sites.
2. COB does not require processes to depend on material entities. This would permit characteristics that don't depend on ICs.

**Comment by cmungall** [comment]
Let's try and keep this issue focused on the general issue at hand

To discuss SDC specifically, use #264 

The fact that names are different is a feature not a bug. It indicates something is not aligned. Make issues for the specific issues, here or in the external ontology linked to here. We will align eventually. Each case will be different.

The GO nucleus one should be an easy one. A non GO person should make an issue
